pkg/move/generate.go

```diff
@@ -0,0 +1,266 @@
+package move
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+)
+
+const (
+	enrichmentDir          = "enrichment"
+	oneAgentDir            = "oneagent/agent/config"
+	containerConfigFile    = "container.conf"
+	metadataJsonFile       = "dt_metadata.json"
+	metadataPropertiesFile = "dt_metadata.properties"
+
+	// all key that come into the metadata files
+	k8sPodNameKey              = "k8s.pod.name"
+	k8sPodUidKey               = "k8s.pod.uid"
+	k8sNamespaceNameKey        = "k8s.namespace.name"
+	k8sClusterUidKey           = "k8s.cluster.uid"
+	k8sWorkloadKindKey         = "k8s.workload.kind"
+	k8sWorkloadNameKey         = "k8s.workload.name"
+	k8sContainerNameKey        = "k8s.container.name"
+	deprecatedClusterUidKey    = "dt.kubernetes.cluster.id"
+	depcrecatedWorkloadKindKey = "dt.kubernetes.workload.kind"
+	depcrecatedWorkloadNameKey = "dt.kubernetes.workload.name"
+
+	// all key that come into the container config file
+	containerConfPodNameKey          = "k8s_fullpodname"
+	containerConfPodUidKey           = "k8s_poduid"
+	containerConfNamespaceNameKey    = "k8s_namespace"
+	containerConfClusterUidKey       = "k8s_cluster_id"
+	containerConfContainerNameKey    = "containerName"
+	containerConfK8sContainerNameKey = "k8s_containername"
+
+	containerImageRegistryKey   = "container_image.registry"
+	containerImageRepositoryKey = "container_image.repository"
+	containerImageTagsKey       = "container_image.tags"
+	containerImageDigestKey     = "container_image.digest"
+)
```

Comment:
why aren't you using `structs` with json tags ([example](https://github.com/Dynatrace/dynatrace-operator/blob/056d80263755460a9a787e261ae72cde1cd6d743/pkg/injection/startup/enrichment.go#L12-L31)) ?

the `--attribute-container` is already a json, you could easily unmarshal it.

you could do similar things with the `--attribute`, you don't get it as json, BUT:
1. just build a `map[string]string` from the splited output
2. marshal that to json
3. unmarshal that to a pretty `struct` that has json tags
- (sidenote: there is a [wellknown library for this](https://pkg.go.dev/github.com/mitchellh/mapstructure), but IMO we don't really need it)

`structs` would nicely group things, they could have methods to help you use them